tests/utils/log/test_es_task_handler.py

```diff
@@ -319,4 +319,4 @@ def test_render_log_id(self):
 
     def test_clean_execution_date(self):
         clean_execution_date = self.es_task_handler._clean_execution_date(self.EXECUTION_DATE)
-        self.assertEqual('2016_01_01T00_12_00_000000', clean_execution_date)
+        self.assertEqual('2016_01_01T00_00_00_000000', clean_execution_date)
```

Comment:
Could we change the test date we use to `datetime(2016, 7, 8, 9, 10, 11, 12)` so that each component has a unique value?

Reply:
fixed.